android/src/main/java/com/reactnativestripesdk/PaymentOptionDisplayDataMapper.kt

```diff
@@ -12,4 +16,29 @@ fun EmbeddedPaymentElement.PaymentOptionDisplayData.toWritableMap(): WritableMap
     putString("label", label)
     putString("paymentMethodType", paymentMethodType)
     putMap("billingDetails", mapFromPaymentSheetBillingDetails(billingDetails))
+
+    // Direct access to imageLoader and call it synchronously with timeout
+    val imageBase64 =
+      try {
+        val imageLoaderField = this@toWritableMap.javaClass.getDeclaredField("imageLoader")
```

Comment:
This might not work if the merchant uses progaurd/R8. 

That being said, I'm not convinced this is the right approach to begin with!

What are you trying to do here?